thoth/package_analyzer/python/fetch.py

```diff
@@ -21,6 +21,7 @@
 
 from thoth.python import Source
 from thoth.package_analyzer.base import FetcherBase
+from thoth.python.exceptions import NotFound
```

Comment:
In `thoth-python` there is a `NotFound` exception https://github.com/thoth-station/python/blob/master/thoth/python/exceptions.py#L52. I'm using that, is that fine?

Reply:
Yes, the exception was designed for exactly this purpose :+1: